src/SprykerEco/Yves/Braintree/Model/Mapper/PaypalResponse/PaypalResponseMapper.php

```diff
@@ -184,7 +184,9 @@ protected function addPaymentTransfer(
         $brainTreePaymentTransfer = new BraintreePaymentTransfer();
         $brainTreePaymentTransfer->setNonce($paypalExpressSuccessResponseTransfer->getNonce());
         $brainTreePaymentTransfer->setBillingAddress($quoteTransfer->getBillingAddress());
-        $brainTreePaymentTransfer->setLanguageIso2Code(mb_strtolower($quoteTransfer->getBillingAddress()->getIso2Code()));
+        if ($quoteTransfer->getBillingAddress()->getIso2Code()) {
```

Comment:
depending on other code is should be either `getBillingAddressOrFail` 
of `if ($quoteTransfer->getBillingAddress() && $quoteTransfer->getBillingAddress()->getIso2Code())`

Reply:
Since initially there were no checks for billing address existence and it would fail anyway for an empty value, theoretically we could use OrFail without a BC break. But since it’s a mapper class, there probably should be no logic containing hard requirements for any properties (as discussed in this PR eariler). So I added an additional check for `$quoteTransfer->getBillingAddress()` to be not empty